### PR TITLE
Update test suit with Environment module

### PR DESCRIPTION
**Pull request by vijay4953**
Updating the test suite with environment module.

### CODE REVIEW

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts [hunk 1]
```diff
@@ -95,183 +108,696 @@ describe('Overview', () => {
...
+            jazzServices_po.gitLogin().click();
+            browser.wait(EC.visibilityOf(jazzServices_po.drpGitBranchType()), timeOutHigh);
+            jazzServices_po.drpGitBranchType().click();
+            jazzServices_po.selectGitBranchType().click();
+            jazzServices_po.gitBranchName().sendKeys(test);
+            browser.wait(EC.elementToBeClickable(jazzServices_po.btnGitCreateBranch()), timeOutHigh);
+            jazzServices_po.btnGitCreateBranch().click();
+            browser.sleep(tenk);
+            //Below code is to write the text into index.js file in SCM for generating logs
+            // jazzServices_po.gitIndexFile().click();
```
Comment (pspani) [inline core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: remove commented line

File: core/jazz_ui/protractor.conf.js [hunk 2]
```diff
@@ -23,6 +25,7 @@ exports.config = {
     'chromeOptions': {
       // Get rid of --ignore-certificate yellow warning
       args: ['--headless', '--disable-gpu', '--no-sandbox', '--test-type=browser'],
+        //  [ , '--disable-gpu', '--no-sandbox', '--test-type=browser'],
```
[the remark this answers is not in the document]
Comment (vijay4953) [reply core/jazz_ui/protractor.conf.js]: removed

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts [hunk 3]
```diff
@@ -38,10 +48,11 @@ describe('Overview', () => {
     jazzServices_po.getNameSpace().sendKeys('jazztest');
     jazzServices_po.getServiceDescription().sendKeys('Testing');
   }
+
   function serviceapprover() {
-    browser.driver.sleep(5000);
+    browser.driver.sleep(fivek);
     jazzServices_po.getSubmit().click();
-    fluentwaittry(jazzServices_po.getDone(), 15000);
+    fluentwaittry(jazzServices_po.getDone(), tenk);
```
[the remark this answers is not in the document]
Comment (vijay4953) [reply core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: yes I have changed to fifteenk

File: core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts (hunk 1 above)
Comment (vijay4953) [reply core/jazz_ui/e2e/04-jazzservices/jazzservices.e2e-spec.ts]: removed

**Pull request merged**